### DISCUSSION TITLE
Configure the event viewer

### DISCUSSION THREAD

**Discussion by joyshmitz** [riclolsen/json-scada#11] [Q&A] [answered]
next question 
Can we configure the event viewer to  see add anotation event at tabular screen viewer for each bay?

**Comment by riclolsen** [comment, marked as answer]
Currently, there is no such option.
Operator actions such as annotations are added to the "userActions" collection on mongodb. This is not currently shown in the UI, it is just for logging actions. Eventually, I may create a UI for this.
However, you can monitor changes to this collection using change streams (e.g. with a nodejs script) and insert a new document into the soeData collection, this way the Events Viewer will show it.

**Reply by joyshmitz** [reply]
this option is very useful for us, I can't say about the rest of the market

**Reply by riclolsen** [reply]
I agree it can be a useful feature. 
Operator UI dev is mostly frozen now until the project enters production quality in about 2 months. Then I will resume working on this kind of feature.
Thank you very much for the suggestion. Interaction and feedback are important for the project.

**Reply by joyshmitz** [reply]
Thank you for the great product, we will work on it together

**Reply by joyshmitz** [reply]
we launch the first working version on windows, I will write how the operation is going

**Reply by riclolsen** [reply]
Blocking annotations now generate events for Events Viewer.
https://github.com/riclolsen/json-scada/blob/master/src/server_realtime_auth/index.js
